app/views/steps/submission/declaration/_initial.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1 class="govuk-heading-xl">
-  Enter your details to confirm the following
+  Submit evidence to confirm the following
```

Comment:
I'm not sure that this is correct. Now the title for all declaration pages whether evidence is expected or not

Reply:
Oh! Sorry. Thanks for spotting it.